app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Base class to be inherited by all Mailer class
+class ApplicationMailer < ActionMailer::Base
+  include ActiveSupport::Rescuable
+  # TODO: get the correct from email address
+  default from: "#{I18n.t('email_from_name')} <notifications@example.com>"
```

Comment:
As `default` is a class level method, this will be evaluated when the class is read at rails boot time. Hence the language will always be the startup language (likely English). I think in our case, it's fine to just have "GoodCity <contact@goodcity.hk>" as the default in a constant.